packages/snaps-cli/src/cmds/init/initHandler.ts

```diff
@@ -58,7 +58,7 @@ export async function initHandler(argv: YargsArgs) {
   }
 
   const directoryToUse = directory
-    ? pathUtils.join(process.cwd(), directory)
+    ? pathUtils.resolve(process.cwd(), directory)
```

Comment:
`mm-snap init` did not support absolute paths before, making it difficult to test.